Don't add include and link paths; MacPorts will set these.
Don't override the install prefix; MacPorts will set this.
--- CMakeLists.txt.orig	2024-01-12 08:08:23.000000000 -0600
+++ CMakeLists.txt	2024-01-24 09:31:06.000000000 -0600
@@ -217,11 +217,6 @@
 set(CMAKE_AUTOUIC ON)
 
 if(APPLE)
-    include_directories("/opt/local/include")
-    link_directories("/opt/local/lib")
-
-    #Without setting this, installation would go into /usr/local, which does not exist per default
-    set(CMAKE_INSTALL_PREFIX "/usr" )
 else(APPLE)
     set(OSS_LIB_PATHS "${CMAKE_INSTALL_FULL_LIBDIR}/oss/lib" "/usr/local/lib${LIB_SUFFIX}/oss/lib" )
 endif(APPLE)
